packages/web-client/app/utils/web3-strategies/layer2-chain.ts

```diff
@@ -124,15 +135,15 @@ export default abstract class Layer2ChainWeb3Strategy
     this.updateWalletInfo([], this.chainId);
   }
 
-  private async refreshBalances() {
-    let raw = await this.getDefaultTokenBalance();
-    this.defaultTokenBalance = new BN(String(raw ?? 0));
+  async refreshBalances() {
```

Comment:
@lukemelia currently we're just redoing the `fetchDepotTask` for this. We could also use the same `getErc20Balance` method that's used in layer 1 to fetch token balances, but it felt better to only use one way of fetching balances.